modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/RasterLayerResponse.java

```diff
@@ -804,6 +804,9 @@ private MosaicElement createMosaic() throws IOException {
                                     // Adding globalRoi to the output
                                     RenderedOp rop = (RenderedOp) mosaic;
                                     rop.setProperty("ROI", in.roi);
+
+                                    mosaic = footprintBehavior.postProcessMosaic(mosaic, in.roi,
```

Comment:
I'm not sure, but what if image bounds contains raster bounds instead? Will we miss post processing?